devel/check-container-build-with-prci.sh

```diff
@@ -117,18 +104,35 @@ function clone-repository
 
 
 [ ! -e .git ] && die "This script should be used from the repository root path"
-[ "${ARTIFACT_HASH}" == "" ] && [ ! -e "artifact-hash.txt" ] && die "No AERTIFACT_HASH variable nor artifact-hash.txt file was specified"
-ARTIFACT_HASH="${ARTIFACT_HASH-"$( cat artifact-hash.txt )"}"
 
-ARTIFACT_HASH="${ARTIFACT_HASH}" download-repo-file-to devel/freeipa-prci.repo
+if [ -n "$ARTIFACT_HASH" -o -e "artifact-hash-txt" ]; then
```

Comment:
Typo: 'artifact-hash-txt' should be 'artifact-hash.txt'

Reply:
Thank you @avisiedo, nice pickup.  I fixed it in the latest push.  Now there is the `ARTIFACT_HASH_FILENAME` "constant" which is used instead of the literal value, to reduce the possibility of errors like this.

Reply:
I like that, removing hardcoded values :)